### PR TITLE
NOJIRA |  uses krogo as micro framework

### PR DESCRIPTION
**Pull request by sumit-dhull**

### CODE REVIEW

**Comment by cherry-rd** [comment]
Test comment!

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,6 @@ go 1.15
```
Comment (ravikiran-zs) [inline go.mod:4]: go module path needs to be in the format <git-repo>/directory https://github.com/sumit-dhull/logisticstest/blob/36cf07d8e054ec8e49efc687ea4924f81680a727/go.mod

**Review by ravikiran-zs — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (sumit-dhull) [reply go.mod:4]: in the latest commit, I have addressed the formatting of go module path

File: configs/.local.env [hunk 2]
```diff
@@ -0,0 +1,6 @@
+DB_HOST=localhost
```
Comment (ravikiran-zs) [inline configs/.local.env:1]: This should not be committed in the repo, should be in `.gitignore`. Rest is fine, you can Merge it with this change

**Review by ravikiran-zs — CHANGES_REQUESTED** [review]

File: configs/.local.env (hunk 2 above)
Comment (sumit-dhull) [reply configs/.local.env]: Latest commits remove `.local.env` and adds the config file to `.gitignore`

**Pull request merged**